tf_encrypted/convert/convert.py

```diff
@@ -351,7 +353,7 @@ def specop_from_numberedscope(scope):
 
 def strip_tensor_info(node_name: str) -> str:
   if node_name.startswith("^"):
-    return n[1:]
+    return node_name[1:]
```

Comment:
@jvmncs does this look correct to you? was an on-the-fly bug fix

Reply:
ya, I made this change as well!